### ISSUE TITLE
Nami Misinterprets  Transfer, Incorrectly Displays no assets

### ISSUE THREAD

**Issue by juanflorez** [input-output-hk/nami#934]
**TLDR**:
I made a transfer out of my account using another wallet, Nami reads the history of transactions as everything left my wallet, so it does not show current assets anymore. I know everything is ok, and my assets are still in my account as confirmed by Lace and Eternl, but Nami will not show them anymore. Even after reinstalling, and re-sync.

**Steps to reproduce:**

1. Have a Ledger wallet added to Nami (_as an additional wallet, DO NOT PUT YOUR seed phrase_)
2. (not sure if needed) Add 5 ADA as collateral
3. In one of the accounts add some ADA, some NFT, and some FT. (Nami has its way of deciding what is what, do not get me started)
4. (optional) do the same using Etrnl or Lace to confirm the assets are there.
5. Using your LEDGER LIVE APP, use the max option to transfer all the ADA out of that account.
6. Check history, check that indeed only the ADA left the account.
7. **---> BUG:** Go to Nami: The account shows no ADA, or ASSETS, the history of transactions according to Nami is that everything was transferred out through a multi-sig transaction.
8. 7 - Expected: ADA should be zero, but all the FT and NFT should still be visible. (as is the case in ETRNL and LACE)

**Expected Behavior:**
Nami should show zero ADA but retain NFT and FT displays (like Lace & Eternl).

**Additional Notes:**
You can verify your assets are safe by checking them in Lace or Eternl wallets.

**Comment by Adaman37** [comment]
@juanflorez It sounds like your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support :[Help Center](https://cardano-zendesk.web.app/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by rhyslbw** [comment]
Can you provide the transaction ID?

**Comment by juanflorez** [comment]
fbf7f9ee928689d3fe19c84d5305f3a072d637ea27393cb73c0a15202f5a0490

**Comment by juanflorez** [comment]
Commenting here maybe to revive the thread

**Comment by rhyslbw** [comment]
Ledger Live is deriving a new change address to return the assets that were part of the UTxOs being spent (as inputs). It's not visible in Nami since it's only tracking address index 0, whereas Lace or Etrnl are able to discover other used addresses within the account. Notice the address is different? To resolve this, send the funds back to the known address in Nami from another wallet and avoid using Ledger Live.

<img width="1331" alt="Screenshot 2024-07-19 at 09 50 55" src="https://github.com/user-attachments/assets/38e6c6fa-87c3-47c2-b029-61a5ca4cda00">
